src/main/kotlin/org/rust/cargo/runconfig/RsCommandConfiguration.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.runconfig
+
+import com.intellij.execution.InputRedirectAware
+import com.intellij.execution.configurations.ConfigurationFactory
+import com.intellij.execution.configurations.LocatableConfigurationBase
+import com.intellij.execution.configurations.RunConfigurationWithSuppressedDefaultDebugAction
+import com.intellij.execution.configurations.RunProfileState
+import com.intellij.openapi.project.Project
+import org.jdom.Element
+import org.rust.cargo.project.model.cargoProjects
+import org.rust.cargo.runconfig.command.workingDirectory
+import java.nio.file.Path
+
+
+abstract class RsCommandConfiguration(
+    project: Project,
+    name: String,
+    factory: ConfigurationFactory
+) : LocatableConfigurationBase<RunProfileState>(project, factory, name),
+    RunConfigurationWithSuppressedDefaultDebugAction,
+    InputRedirectAware.InputRedirectOptions {
```

Comment:
Looks like `InputRedirectAware.InputRedirectOptions` doesn't make sense for `WasmPackCommandConfiguration`. So let's move all implementation details related to `InputRedirectAware.InputRedirectOptions` into `CargoCommandConfiguration`